### DISCUSSION TITLE
Depolar gizlemek

### DISCUSSION THREAD

**Discussion by Mhtasf63** [community/community#108292] [Programming Help] [answered]
### Body

Gethup gizlilik için otomatik ayar linki

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Mhtasf63 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
